test/functional/signature/cert-chain.bats

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env bats
```

Comment:
I can tell you are not using the `generate_test [relative_path]/<test_name>` command from testlib to generate all these new tests since they are missing the header boilerplate:
```
# Author: Otavio Pontes
# Email: otavio.pontes@intel.com
```
You may want to consider using it in the future, it saves you time by creating a new test file with the appropriate test ID, just saying... 😉

Reply:
I'll fix that, but copy and paste is a way easier way to create tests than the script.
Btw, I don't like having people names in files, why do we really needs it?
Because we use git as the source control and the information on who created and changed the file is always available in git.

Reply:
And only 40% of our tests have author information in it. So I don't think that this information is going to be useful

Reply:
> I'll fix that, but copy and paste is a way easier way to create tests than the script.

True, but it is easier to copy mistakes that existed in the previous tests too, that way you always start from scratch with a template. But that is just my opinion, if you like copy/paste by all means go ahead 😁 


> Btw, I don't like having people names in files, why do we really needs it?

Yeah, I don't like having names on source file either, but the idea of having them in tests was just because sometimes tests can be confusing since some tests are meant to validate corner cases or weird scenarios difficult to understand by reading the code. So it was an easier way to contact the author in case you need help with the tests. But we can get rid of the name on tests if we do a good job documenting what the test is attempting to test within the test itself.
I know we can use git blame but some times it is difficult to see the real author when many people have modified the test file. But maybe is just me not being too familiar with that command. 😝 

Bottom line, adding the name is optional and we can get rid of names all together if you think they are not useful.